src/robotka.h

```diff
@@ -184,7 +186,145 @@ void rkMotorsSetSpeedRight(int8_t speed);
  * \param id číslo motoru od 1 do 4 včetně
  * \param speed rychlost motoru od od -100% do 100%
  */
-void rkMotorsSetSpeedById(int id, int8_t speed);
+void rkMotorsSetSpeedById(uint8_t id, int8_t speed);
+
+/**
+ * \brief Popojetí oběma motory o vzdálenost v mm (blokující).
+ *
+ * Tato funkce zablokuje program, dokud robot danou vzdálenost neujede.
+ *
+ * \param mmLeft kolik milimetrů má ujet levý motor
+ * \param mmRight kolik milimetrů má ujet pravý motor
+ * \param speed rychlost, kterou má robot jet od -100% do 100%
+ */
+void rkMotorsDrive(float mmLeft, float mmRight, uint8_t speed);
+
+/**
+ * \brief Popojetí levým motorem o vzdálenost v mm (blokující).
+ *
+ * Tato funkce zablokuje program, dokud robot danou vzdálenost neujede.
+ *
+ * \param mm kolik milimetrů má ujet levý motor
+ * \param speed rychlost, kterou má motor jet od -100% do 100%
+ */
+void rkMotorsDriveLeft(float mm, uint8_t speed);
+
+/**
+ * \brief Popojetí pravým motorem o vzdálenost v mm (blokující).
+ *
+ * Tato funkce zablokuje program, dokud robot danou vzdálenost neujede.
+ *
+ * \param mm kolik milimetrů má ujet pravý motor
+ * \param speed rychlost, kterou má motor jet od -100% do 100%
```

Comment:
Kolik tiků za sekundu je rovno 100 % `speed` je definováno v `rkConfig& cfg`?

Reply:
Nastřeleno na 2000, nedá se konfigurovat momentálně, volno k diskusi. Zase dokud máme pouze žluté motory, tak bych to do rkConfig nedával.

Reply:
Pokud se ty motory značně neliší, tak proč ne, ale pokud jsem dobře pochopil tu fotku na FB kde byly rozdělený motory podle otáček od 100 do 200 tak tam může být docela rozdíl. Pak to totiž může dělat problém i pro regulátory (když motor pojede na max ale regulátor se bude snažit ještě přidávat - pokud tedy máme PID, ne jen P reg,).